ios/Podfile

```diff
@@ -6,19 +6,14 @@ platform :ios, '10.0'
 target 'CalendarsExample' do
   config = use_native_modules!
   
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # to enable hermes on iOS, change `false` to `true` and then install pods
+    :hermes_enabled => false
+  )
 
   target 'CalendarsExampleTests' do
     inherit! :search_paths
     # Pods for testing
   end
 end
-
-target 'CalendarsExample-tvOS' do
-  # Pods for CalendarsExample-tvOS
-
-  target 'CalendarsExample-tvOSTests' do
-    inherit! :search_paths
-    # Pods for testing
-  end
-end
```

Comment:
In the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2) they're adding the following code as well: 

```
use_flipper!()

post_install do |installer|
  react_native_post_install(installer)
```

Reply:
I'm not using flipper in calendars...